runtime/sunvmi/sunvmi.c

```diff
@@ -217,7 +217,29 @@ getCallerClassJEP176Iterator(J9VMThread * currentThread, J9StackWalkState * walk
 
 	Assert_SunVMI_mustHaveVMAccess(currentThread);
 
-	if (J9_ARE_ALL_BITS_SET(J9_ROM_METHOD_FROM_RAM_METHOD(walkState->method)->modifiers, J9AccMethodFrameIteratorSkip)) {
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE) && (JAVA_SPEC_VERSION == 11)
+/* Do not skip InjectedInvoker classes despite them having the J9AccMethodFrameIteratorSkip
+ * modifier set via the @Hidden attribute. Skipping them causes incorrect, unexpected
+ * behaviour when using OpenJDK method handles pre-hidden-class support.
+ */
```

Comment:
Indentation missing.
```suggestion
	/* Do not skip InjectedInvoker classes despite them having the J9AccMethodFrameIteratorSkip
	 * modifier set via the @Hidden attribute. Skipping them causes incorrect, unexpected
	 * behaviour when using OpenJDK method handles pre-hidden-class support.
	 */

	
```